addons/l10n_ar_withholding/tests/test_withholding_ar_ri.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.exceptions import UserError
+from odoo.tests import tagged, Form
+from odoo import Command
+from odoo.tests.common import TransactionCase, Form
+
+
+@tagged('post_install', '-at_install')
+class TestL10nArWithholdingArRi(AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(cls, chart_template_ref='ar_ri'):
+
+        super().setUpClass(chart_template_ref=chart_template_ref)
+
+        cls.tax_wth_seq_test_1 = cls.env['ir.sequence'].create({
+        'implementation': 'standard',
```

Comment:
indent
```suggestion
            'implementation': 'standard',
```

Reply:
@maq-adhoc is this ready to be closed? As I see this is Outdated.
cc' @jco-odoo